serverless/src/submitSwodlrOrder/__tests__/mocks.js

```diff
@@ -0,0 +1,13 @@
+export const mockSwodlrOrder = {
```

Comment:
I don't think we need/want to check in this file. On the unit tests we can just call the handler with an object as I think you are already doing, this can be used for testing in your `/tmp` file or something uncommitted to git

Reply:
I'm not sure what you mean. This object is critical for the test I think. We can make it a const but we have that sort of file in other submit tests.

Reply:
No you are right I thought this wasn't getting used